templates/microsoft.storage/storageaccounts/metrics.json

```diff
@@ -0,0 +1,41 @@
+[
+    {
+        "metricName": "Availability",
+        "description": "Availability of a storage account.",
+        "reference": "https://link/to/more/information",
+        "aggregationType": "Minimum",
+        "timeGrain": "PT15M",
+        "degradedThreshold": "100",
+        "degradedOperator": "LowerThan",
+        "unhealthyThreshold": "99",
+        "unhealthyOperator": "LowerThan",
+        "dataUnit": "Percent",
+        "recommended": "true"
+    },
+    {
+        "metricName": "SuccessE2ELatency",
+        "description": "SuccessE2ELatency of a storage account.",
+        "reference": "https://link/to/more/information",
+        "aggregationType": "Average",
+        "timeGrain": "PT5M",
+        "degradedThreshold": "10",
+        "degradedOperator": "GreaterThan",
+        "unhealthyThreshold": "15",
+        "unhealthyOperator": "GreaterThan",
+        "dataUnit": "MilliSeconds",
+        "recommended": "false"
```

Comment:
Why not make this recommended as well?

Reply:
Not sure about the value / what this really means and if the thresholds are viable. Let's figure that out first. We can easily adjust that - and will probably do that in the next couple of weeks as soon as users get their hands on it.